src/main/java/org/opentripplanner/street/model/vertex/StreetVertex.java

```diff
@@ -72,4 +75,29 @@ public boolean isConnectedToDriveableEdge() {
   public boolean isEligibleForCarPickupDropoff() {
     return isConnectedToDriveableEdge() && isConnectedToWalkingEdge();
   }
+
+  /**
+   * Returns the list of area stops that this vertex is inside of.
+   */
+  @Nonnull
+  public Set<AreaStop> areaStops() {
+    return Objects.requireNonNullElseGet(areaStops, Set::of);
```

Comment:
Can the return value be cached? Otherwise, it seems that a new set will be created on each call.

Reply:
Disregard the above comment.